### PR TITLE
policy-daemon: invalidate cache on file move too

### PR DESCRIPTION
**Pull request by marmarek**
Invalidate cache not only when file is modified inside policy dir, but
also when it's moved in or out.

### CODE REVIEW

**Comment by DemiMarie** [comment]
Looks geat, except there are no tests :slightly_smiling_face:.

File: qrexec/tests/policy_cache.py [hunk 1]
```diff
@@ -97,6 +97,47 @@ async def test_13_no_change(self, tmp_path, mock_parser):
...
+        file_moved.rename(file)
+
+        await asyncio.sleep(1)
+
+        assert cache.outdated
+
+        cache.get_policy()
+
+        call = unittest.mock.call(policy_path=policy_path)
+        assert mock_parser.mock_calls == [call, call, call]
```
Comment (DemiMarie) [inline qrexec/tests/policy_cache.py:139]: What about moving within the directory?

**Review by DemiMarie — COMMENTED** [review]
One more test that could be added, but IIUC it won’t add new coverage.

File: qrexec/tests/policy_cache.py (hunk 1 above)
Comment (marmarek) [reply qrexec/tests/policy_cache.py:139]: It's then both move from and move to, so this case is covered too.

**Pull request merged**